### PR TITLE
Metamodel 179

### PR DESCRIPTION
**Pull request by tomaszguzialek**

### CODE REVIEW

File: hadoop/src/test/java/org/apache/metamodel/util/HdfsResourceIntegrationTest.java [hunk 1]
```diff
@@ -1,187 +1,244 @@
...
+                _port = Integer.parseInt(portString);
+            }
+        } else {
+            configured = false;
+        }
+        Assume.assumeTrue(configured);
+    }
+
+    @After
+    public void tearDown() throws Exception {
```
Comment (tomaszguzialek) [inline hadoop/src/test/java/org/apache/metamodel/util/HdfsResourceIntegrationTest.java]: Diff not visible, so marking the place where changes are. Added a tearDown method to make sure we don't leave our test file in the cluster.

File: hadoop/src/test/java/org/apache/metamodel/util/HdfsResourceIntegrationTest.java [hunk 2]
```diff
@@ -1,187 +1,244 @@
...
+        logger.info(stopwatch.elapsed(TimeUnit.MILLISECONDS) + " - deleted");
+
+        Assert.assertFalse(res1.isExists());
+
+        logger.info(stopwatch.elapsed(TimeUnit.MILLISECONDS) + " - done");
+        stopwatch.stop();
+    }
+
+    @Test
+    public void testFileSystemNotBeingClosed() throws IOException {
```
Comment (tomaszguzialek) [inline hadoop/src/test/java/org/apache/metamodel/util/HdfsResourceIntegrationTest.java]: Diff not visible, so marking the place where changes are. Added a test case.

**Comment by LosD** [comment]
Looks good to me

**Comment by tomaszguzialek** [comment]
I had to remove tearDown method and put the test in try-finally block to make the build work.

**Comment by kaspersorensen** [comment]
+1 from me too

**Comment by kaspersorensen** [comment]
merging

**Pull request closed without merging**